cpp/velox/memory/VeloxMemoryManager.cc

```diff
@@ -163,31 +138,18 @@ VeloxMemoryManager::VeloxMemoryManager(
   glutenAlloc_ = std::make_unique<ListenableMemoryAllocator>(allocator.get(), listener_.get());
   arrowPool_ = std::make_unique<ArrowMemoryPool>(glutenAlloc_.get());
 
-  auto veloxAlloc = velox::memory::MemoryAllocator::getInstance();
-
-#ifdef GLUTEN_ENABLE_HBM
-  wrappedAlloc_ = std::make_unique<VeloxMemoryAllocator>(allocator.get(), veloxAlloc);
-  veloxAlloc = wrappedAlloc_.get();
-#endif
-
   ArbitratorFactoryRegister afr(listener_.get());
   velox::memory::MemoryManagerOptions mmOptions{
-      velox::memory::MemoryAllocator::kMaxAlignment,
-      velox::memory::kMaxMemory,
-      velox::memory::kMaxMemory,
-      true, // memory usage tracking
-      true, // leak check
-      false, // debug
-      false, // coreOnAllocationFailureEnabled
-#ifdef GLUTEN_ENABLE_HBM
-      wrappedAlloc.get(),
-#else
-      veloxAlloc,
-#endif
-      afr.getKind(),
-      0,
-      32 << 20,
-      0};
+      .alignment = velox::memory::MemoryAllocator::kMaxAlignment,
```

Comment:
@zhztheplayer Could you help to review these changes? Thanks!

Reply:
LGTM. Thanks!